### DISCUSSION TITLE
Is there a reason the TrackID column is reversed by default?

### DISCUSSION THREAD

**Discussion by J-Cake** [kraxarn/spotify-qt#46] [Q&A] [answered]
Perhaps the first thing I noticed was that the TrackID Column is inverted by default. I'm wondering if this is intentional and/or how to change it?

Thanks

**Comment by kraxarn** [comment, marked as answer]
Oops, forgot discussions are a thing now :sweat_smile:

It's not really intentional, it just previously sorted by track name, where it made sense to sort in that order and it wasn't updated when track numbers were added. I have just always sorted it by artist (you can change on what to order by if you right-click the header, which hasn't been fully updated either, but uncheck all to sort by track number).
Track numbers being sorted in the wrong order by default isn't intended and will be fixed for the next release though.

**Reply by J-Cake** [reply]
I SAW THE RELEASE NOTES!!!! I WAS BEAMING!!!

**Reply by J-Cake** [reply]
Also, I used a bit of a hack: I said that the default column should be the date added column. That worked...
